### PR TITLE
ENT-9129: Initialize SSL related items in cf-testd to avoid segfault on aarch64

### PR DESCRIPTION
**Pull request by craigcomstock**
Ticket: ENT-9129
Changelog: title

### CODE REVIEW

**Comment by craigcomstock** [comment]
@cf-bottom jenkins please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8584/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8584/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8584/

**Comment by craigcomstock** [comment]
This failure is actually a step forward! Both arm failures are from https://github.com/cfengine/masterfiles/pull/2483 so I think after that one is merged we can re-check this one and be done :)

File: cf-testd/cf-testd.c [hunk 1]
```diff
@@ -774,6 +774,7 @@ int main(int argc, char *argv[])
     for (int i = 0; i < n_threads; i++)
     {
         thread_configs[i] = (CFTestD_Config*) xmalloc(sizeof(CFTestD_Config));
+        memset(&thread_configs[i], 0, sizeof(CFTestD_Config));
```
Comment (vpodzime) [inline cf-testd/cf-testd.c:777]: It's easier to replace the `xmalloc()` call with `xcalloc()`.

Comment (craigcomstock) [reply cf-testd/cf-testd.c:777]: agreed, will change.

**Review by vpodzime — DISMISSED** [review]
ACK

**Review by vpodzime — APPROVED** [review]
ACK

**Comment by craigcomstock** [comment]
@cf-bottom jenkins please, now that https://github.com/cfengine/masterfiles/pull/2483 is merged I expect that arm64 ubu22 builds will be green! :O

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8590/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8590/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8590/

**Comment by craigcomstock** [comment]
cherry picked to 3.18.x at https://github.com/cfengine/core/pull/5049

**Comment by craigcomstock** [comment]
only php selenium tests failed so merged.

**Pull request merged**